### PR TITLE
Fixes after trying it out

### PR DESCRIPTION
**Pull request by balsoft**

### CODE REVIEW

File: haskell.nix/flake.nix [hunk 1]
```diff
@@ -5,7 +5,7 @@
       flake = false;
     };
     nixpkgs.url = "github:serokell/nixpkgs";
-    haskell-nix.url = "github:serokell/haskell.nix/serokell-latest";
+    haskell-nix.url = "github:input-output-hk/haskell.nix";
```
Comment (mkaito) [inline haskell.nix/flake.nix:8]: Context?

File: haskell.nix/flake.nix [hunk 2]
```diff
@@ -15,102 +15,127 @@
...
+    # url = "github:serokell/haskell-nix-weeder";
+    # flake = false;
+    #};
   };
 
-  outputs = { self, nixpkgs, haskell-nix, hackage-nix, stackage-nix, ... }:
+  outputs =
+    { self, nixpkgs, haskell-nix, hackage-nix, stackage-nix, flake-utils, ... }:
+    ## for weeder:
+    # { self, nixpkgs, haskell-nix, hackage-nix, stackage-nix, flake-utils, haskell-nix-weeder, ... }:
```
Comment (mkaito) [inline haskell.nix/flake.nix]: Wouldn't it be less convoluted to just `@args` here, and then just have a snippet below using that?

File: haskell.nix/flake.nix [hunk 3]
```diff
@@ -15,102 +15,127 @@
...
-
-        # runs the test
-        test = hs-pkg.checks.pataq-test;
-      };
-
-      # script for running weeder
-      ##weeder-script = weeder-script;
-    };
+          # runs the tests
+        } // hs-checks;
```
Comment (mkaito) [inline haskell.nix/flake.nix:133]: Should we `// self.packages.${system}` here as well, to build everything with `nix flake check` in a single step?

**Review by mkaito — APPROVED** [review]

File: haskell.nix/flake.nix (hunk 1 above)
Comment (balsoft) [reply haskell.nix/flake.nix:8]: Our fork is currently broken.

File: haskell.nix/flake.nix (hunk 2 above)
Comment (balsoft) [reply haskell.nix/flake.nix]: Yes, it would. Feel free to push this :)

File: haskell.nix/flake.nix (hunk 3 above)
Comment (balsoft) [reply haskell.nix/flake.nix:133]: There's already `checks.${system}.build-all`

File: haskell.nix/flake.nix (hunk 1 above)
Comment (mkaito) [reply haskell.nix/flake.nix:8]: Are we going to fix it?

**Pull request closed without merging**